app/src/main/res/layout/fragment_client_settings.xml

```diff
@@ -57,6 +58,34 @@
             app:layout_constraintTop_toBottomOf="@+id/childNameEtGLTop"
             />
 
+    <RadioGroup
+            android:id="@+id/voiceAnalysisRadioButtons"
+            android:layout_marginTop="@dimen/spacing_small"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintEnd_toEndOf="@id/clientSettingsGLEnd"
+            app:layout_constraintStart_toStartOf="@id/clientSettingsGLBegin"
+            app:layout_constraintTop_toBottomOf="@id/childNameEtGLBottom"
+            >
+
+        <androidx.appcompat.widget.AppCompatRadioButton
+                android:id="@+id/noiseDetectionOption"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:text="Noise Detection"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/89ff9ffb603a83b79e177dd00519720c6a8d52ac/app/src/main/res/layout/fragment_client_settings.xml#L77"></span>Hardcoded string “Noise Detection”, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>